src/components/MenuItem.js

```diff
@@ -124,11 +124,15 @@ const MenuItem = ({
                         </View>
                     )}
                     <View style={[styles.justifyContentCenter, styles.menuItemTextContainer]}>
-                        <Text style={[styles.createMenuText, styles.ml3, (disabled ? styles.disabledText : undefined)]}>
+                        <Text style={[
+                            styles.popoverMenuText,
+                            styles.ml3, (disabled ? styles.disabledText : undefined),
```

Comment:
NAB would prefer to see one style per line when we're over the max limit for a single line:

```suggestion
                            styles.ml3,
                            (disabled ? styles.disabledText : undefined),
```